.github/actions/docker-build-and-push/action.yaml

```diff
@@ -54,6 +54,17 @@ runs:
         echo "$EOF" >> $GITHUB_OUTPUT
       shell: bash
 
+    - name: Docker meta for base
+      id: meta-base
+      uses: docker/metadata-action@v5
+      with:
+        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.bake-target }}
+        tags: ${{ steps.set-docker-tags.outputs.tags }}
+        bake-target: docker-metadata-action-base
+        flavor: |
+          latest=false
+          suffix=-base
```

Comment:
https://github.com/autowarefoundation/autoware/pull/4849#discussion_r1639516223 @youtalk @oguzkaganozt I've read this, that's I did not make this like the other 3 below:
- `suffix=-base${{ inputs.tag-suffix }}`
- and just made it `suffix=-base`

Please let me know if what I did is alright, I don't know what any of these tags are for.

![image](https://github.com/autowarefoundation/autoware/assets/10751153/a136fc7f-31e0-4643-9b6b-b9e516e4b3e5)

Reply:
I think `suffix=-base` is OK because it has no CUDA requirements.